README.md

```diff
@@ -131,8 +131,6 @@ To add test data to the app, do the following:
 
 - Add the following to your hosts file. Path to file on mac: /private/etc/hosts. Path to file on Windows: c:\Windows\System32\Drivers\etc\hosts.
 ```#Subdomain for accessmanagement
-127.0.0.1 am.ui.local.altinn.cloud #ipv4
-0000:0000:0000:0000:0000:0000:0000:0001 am.ui.local.altinn.cloud #ipv6
```

Comment:
And the part of adding to the host file.
Cause now, it says to add something to the file, but not what to add 😅

Reply:
Removing it because it's a duplicate line.

It says like this, I think it's quite clear what's supposed to be in the file. If you click on the 3 dots on the file and click view file you could've seen it.

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/99e1a031-9221-4d19-b97c-64e4046219d8)

Reply:
Ahh, ok, I didn't see that.
Never mind then!